src/qnngds/cells.py

```diff
@@ -578,18 +581,17 @@ def snspd(
     text: Union[None, str] = dflt.text,
     fill_pad_layer: bool = False,
 ) -> Device:
-    """Creates a cell that contains a vertical superconducting nanowire single-
-    photon detector (SNSPD).
+    """Creates a cell that contains vertical superconducting nanowire single-
+    photon detectors (SNSPD).
 
     Parameters:
         die_w (int or float): Width of a unit die/cell in the design (the output
         device will be an integer number of unit cells).
```

Comment:
This text ("_device will be an integer number of unit cells)._") should not be on a new line! (it messes up with the markdown on the API.)